lib/tasks/app_initializer.rake

```diff
@@ -12,6 +12,7 @@ namespace :app_initializer do
     Rake::Task["data_updates:enqueue_data_update_worker"].execute
 
     puts "\n== Bust Caches =="
+    Rake::Task["cache:clear_app_cache"].execute unless Rails.env.production?
```

Comment:
Since E2E tests run "production" like, it probably makes sense adding the E2E environment to the `unless`?

Thoughts?

Reply:
> Since E2E tests run "production" like

Sorry, but you mean on Travis, right? The `Rails.env` for that should still be test, that's the global value defined in `.travis.yml`:

```yml
env:
  global:
    - RAILS_ENV=test
```

The two E2E nodes do not override this, only the asset precompilation task does:

```yml
        - NODE_ENV=production RAILS_ENV=production bundle exec rake assets:precompile
```